### DISCUSSION TITLE
AI Assist to find or generate custom templates.

### DISCUSSION THREAD

**Discussion by samyakkkk** [projectdiscovery/nuclei-templates#10177] [Ideas]
Hi there, 

I'm helping open-source projects build AI assists trained on their repository and documentation. 

Came across this project and had an instant idea of creating an AI assist where user can custom generate templates for them based on existing templates. 

Could this be of interest to the users? If so, I can quickly come up with something. (also it's free for end users)

**Comment by GeorginaReeder** [comment]
Hi @samyakkkk - thanks so much for your suggestion here.

I'm not sure if AI assists are something we're actively looking into currently. @ehsandeep do you have any thoughts on this?

**Comment by ehsandeep** [comment]
@samyakkkk thanks for offering help! but we already got this done, please refer to https://cloud.projectdiscovery.io/templates to see it in action and https://docs.projectdiscovery.io/cloud/editor/overview for the docs.
